lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -0,0 +1,408 @@
+
+# This code is part of Ansible, but is an independent component
+
+# This particular file snippet, and this file snippet only, is BSD licensed.
+# Modules you write using this snippet, which is embedded dynamically by Ansible
+# still belong to the author of the module, and may assign their own license
+# to the complete work.
+
+# Copyright: (c) 2018, Kevin Breit <kevin.breit@kevinbreit.net>
+# All rights reserved.
+
+# Redistribution and use in source and binary forms, with or without modification,
+# are permitted provided that the following conditions are met:
+#
+#    * Redistributions of source code must retain the above copyright
+#      notice, this list of conditions and the following disclaimer.
+#    * Redistributions in binary form must reproduce the above copyright notice,
+#      this list of conditions and the following disclaimer in the documentation
+#      and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+# IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import os
+from ansible.module_utils.basic import AnsibleModule, json, env_fallback
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils._text import to_native, to_bytes, to_text
+
+
+def meraki_argument_spec():
+    return dict(auth_key=dict(type='str', no_log=True, fallback=(env_fallback, ['MERAKI_KEY'])),
+                host=dict(type='str', default='api.meraki.com'),
+                use_proxy=dict(type='bool', default=False),
+                use_https=dict(type='bool', default=True),
+                validate_certs=dict(type='bool', default=True),
+                output_level=dict(type='str', default='normal', choices=['normal', 'debug']),
+                timeout=dict(type='int', default=30),
+                org_name=dict(type='str', aliases=['organization']),
+                org_id=dict(type='str'),
+                )
+
+
+class MerakiModule(object):
+
+    def __init__(self, module, function=None):
+        self.module = module
+        self.params = module.params
+        self.result = dict(changed=False)
+        self.headers = dict()
+        self.function = function
+
+        if module.params['auth_key'] is None:
+            module.fail_json(msg='Meraki API key not specified')
+        else:
```

Comment:
There's no need to do the 'else', because everything that follows is also your 'else'. It's the continuation of your program.